Semantics.md

```diff
@@ -683,3 +683,22 @@ outside the range which rounds to an integer in range) traps.
 [future types]: FutureFeatures.md#more-table-operators-and-types
 [future large pages]: FutureFeatures.md#large-page-support
 [future ieee 754]: FutureFeatures.md#full-ieee-754-2008-conformance
+
+
+## Validation
+
+A module binary has to be _validated_ before it is compiled.
+Validation ensures that the module is well-defined and that its code cannot exhibit any undefined behavior.
+In particular, along with some runtime checks, it ensures that no program can access or corrupt memory it does not own.
```

Comment:
s/it ensures/this ensures/

Otherwise the two its in the last clause could be confusing.